lib/index.js

```diff
@@ -18,7 +18,7 @@ import badge from './badge'
 import splash from './splash'
 import iframe from './iframe'
 import log from './log'
-import coc from './coc'
+import cocc from './coc'
```

Comment:
coc uparxei hdh san var pio katw, einai mallon auto pou xrhsimopoiei autos gia to coc url

Reply:
εγω το είχα βάλει το `import coc` εδώ. :)

Reply:
Ξερω ναι :P αλλα θα επερνε το var του αλλου πρωτα :)